pkg/securityscan/core/configmap.go

```diff
@@ -36,35 +39,53 @@ func NewConfigMaps(clusterscan *cisoperatorapiv1.ClusterScan, clusterscanprofile
 	}
 	configcm, err := generateConfigMap(clusterscan, "cisscanConfig.template", "./pkg/securityscan/core/templates/cisscanConfig.template", configdata)
 	if err != nil {
-		return configmaps, err
+		return cmMap, err
 	}
+	cmMap["configcm"] = configcm
+
+	var isCustomBenchmark bool
+	var customBenchmarkConfigMapData map[string]string
```

Comment:
This is a nil map so if anything tries to write to it later there will be a panic. Initialize the map with make even if it's going to get replaced so it doesn't cause an issue later.